custom_manifests/manifests/05-oci-eval-user-data.yml

```
@@ -0,0 +1,83 @@
+# ----- 05-oci-eval-user-data.yml -----
+
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 00-master-oci-eval-user-data
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    storage:
+      files:
+        - contents:
+            compression: gzip
+            source: data:;base64,H4sIAAAAAAAC/3SRQY/aMBCF7/4VD5cDHIyBBqSicGgPqIeK/gQ0SSaNpcROHSeCtvz3KrBhw2r3lEjjed97bz5NdNt4nRir2XZIqCmEaDhAcXt2qE3NOZlSCJNjgrZhf8oo0CnZRvvpLG19CfUd8msbCufNHwrG2R2+MXn2+OkpLVlCqYrOKpiK8XkJpTwHf8Fm+FMZl3QZjVTqrPWctw1nUD9QhFDvtF5tvyzWm2jx8tWuTnW31sY2gWzKuuJAvTn9sDkXoWArAE4LB3kgU3KG4NBjDHd8S4T+KXLvKgxaGLRkv3w2ASuRm1sLvzzXUL8ho2WEows4uNZmEnEcQ07/PlV0lc8Gjm4EpI5MSUnJD8ZyYIya3k9nt11l31HHv/5ivI2gVMapy/jDxPfxK/5NsP7wiO+oMecq5+J/AAAA//+8ajHdJAIAAA==
+          mode: 493
+          path: /usr/local/bin/oci-eval-user-data.sh
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Evaluate user data
+            ConditionFirstBoot=yes
+            After=NetworkManager.service
+            Before=ovs-configuration.service kubelet.service
+
+            [Service]
+            ExecStart=/usr/local/bin/oci-eval-user-data.sh
+            Type=oneshot
+            RemainAfterExit=yes
+            Restart=on-failure
+            RestartSec=5
+
+            [Install]
+            WantedBy=multi-user.target
+          enabled: true
+          name: oci-eval-user-data.service
+
+---
+
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 00-worker-oci-eval-user-data
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    storage:
+      files:
+        - contents:
+            compression: gzip
+            source: data:;base64,H4sIAAAAAAAC/3SRQY/aMBCF7/4VD5cDHIyBBqSicGgPqIeK/gQ0SSaNpcROHSeCtvz3KrBhw2r3lEjjed97bz5NdNt4nRir2XZIqCmEaDhAcXt2qE3NOZlSCJNjgrZhf8oo0CnZRvvpLG19CfUd8msbCufNHwrG2R2+MXn2+OkpLVlCqYrOKpiK8XkJpTwHf8Fm+FMZl3QZjVTqrPWctw1nUD9QhFDvtF5tvyzWm2jx8tWuTnW31sY2gWzKuuJAvTn9sDkXoWArAE4LB3kgU3KG4NBjDHd8S4T+KXLvKgxaGLRkv3w2ASuRm1sLvzzXUL8ho2WEows4uNZmEnEcQ07/PlV0lc8Gjm4EpI5MSUnJD8ZyYIya3k9nt11l31HHv/5ivI2gVMapy/jDxPfxK/5NsP7wiO+oMecq5+J/AAAA//+8ajHdJAIAAA==
+          mode: 493
```

does this have to be decimal? would be easier to understand as octal:
```suggestion
          mode: 0755
```

---

[Ignition specifications](https://coreos.github.io/ignition/configuration-v3_4/) say:
> Note that the mode must be properly specified as a decimal value (i.e. 0644 -> 420)